<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmallKeyboardWidget</class>
 <widget class="QWidget" name="SmallKeyboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{font-family: Tamworth Gothic}
QWidget[objectName|=&quot;invisibleButton&quot;]{border: none; }</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>406</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Keyboard Background.png</pixmap>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_backspace" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>131</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_space" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>391</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_shift" native="true">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>131</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>491</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tamworth Gothic</family>
     <pointsize>25</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InvisibleButton</class>
   <extends>QWidget</extends>
   <header>invisiblebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
